src/RoleObjects/MathData.js

```diff
@@ -1,12 +1,11 @@
-import _ from 'lodash';
 import SectionData from './SectionData';
 
 export default class MathData extends SectionData {
   set mathML(mathML) {
-    this._reactProps.dangerouslySetInnerHTML = { __html: mathML };
+    this.text = mathML;
```

Comment:
- not sure this is the proper use of this property @mmoss thoughts?
`* string that describes the DisplayObject for the AT`

Reply:
I'm not sure there is a proper way to to handle this right now. It's not the text, but previously it'd just have returned the raw HTML string, so functionally I don't see how this would be different.

Reply:
@mmoss In Both versions, getters returns the same HTML String. Added an `_htmlString` property in `AccessibilityObject` to store the mathML string, which is later used in AccessibilityTranslator to add the htmlString as child of the element.